vimrc.bundles

```diff
@@ -2,36 +2,51 @@ if &compatible
   set nocompatible
 end
 
-filetype off
-set rtp+=~/.vim/bundle/Vundle.vim/
-call vundle#begin()
+" Shim command and function to allow migration from Vundle to vim-plug.
+function! PluginToPlug(arg, ...)
+  echom "You are using Vundle's `Plugin` command to declare plugins. Dotfiles now uses vim-plug for plugin mangagement. Please rename uses of `Plugin` to `Plug`. Plugin was '".a:arg."'."
+  let vim_plug_options = {}
 
-" Let Vundle manage Vundle
-Plugin 'gmarik/Vundle.vim'
+  if a:0 > 0
+    if has_key(a:1, 'name')
+      let name = a:1.name
+      let vim_plug_options.dir = "$HOME/.vim/bundle/".a:1.name
+    endif
+
+    if has_key(a:1, 'rtp')
+      let vim_plug_options.rtp = a:1.rtp
+    endif
+  endif
+
+  Plug a:arg, vim_plug_options
+endfunction
+
+com! -nargs=+  -bar Plugin call PluginToPlug(<args>)
+
+call plug#begin('~/.vim/bundle')
```

Comment:
I don't think we should move the plugins around. Moving them might introduce a hitch into the transition, and there's no big benefit in having them in a new folder. I think a Ruby developer who is new to Vim might also find the folder `bundle` to be more familiar if they are starting to dig through dotfiles.

Also, pathogen uses `bundle`, so that seems to be a more general practice than just Vundle.